src/System-FileRegistry/GZipReadStream.extension.st

```diff
@@ -12,6 +12,20 @@ GZipReadStream class >> fileReaderServicesForFile: fullName suffix: suffix [
 	^ services
 ]
 
+{ #category : '*System-FileRegistry' }
+GZipReadStream class >> openWithContents: contentsString label: titleString [
+	"Open a text viewer on contentsString and window label titleString"
+
+	SpTextPresenter new
```

Comment:
This should be packaged in a separate package like System-Spec because now we are introducing a dependency from GZip to Spec.

Reply:
Yes I meant from the package that contains the extension to Spec, so System-FileRegistry to Spec.